packaging/generic-deb/debian/README.source

```diff
@@ -0,0 +1,14 @@
+Steps to build and install package:
+
+1. Install build dependencies
+    sudo apt-get update -y && sudo apt-get install -y devscripts build-essential lintian git curl golang-go debhelper
+
+2. Build the package by running
+    make clean deb
+
+3. Install the package with
+    sudo dpkg -i amazon-ecs-init_1.45.0-1_amd64.deb
```

Comment:
Modern versions of apt will handle local files, and since apt resolves dependencies, it'd be preferable to use

     sudo apt install ./amazon-ecs-init_1.45.0-1_amd64.deb

here. Note that the leading `./` is important.

Reply:
done